src/libs/E2E/tests/linkingTest.e2e.ts

```diff
@@ -61,16 +59,7 @@ const test = (config: NativeConfig) => {
                 return;
             }
 
-            if (entry.name === CONST.TIMING.REPORT_INITIAL_RENDER) {
```

Comment:
@kirillzyusko how do you see this part of the test as we're deprecating `report_initial_render`?

I left the `open_report` 'as is' (replacing `switch_report`), but not sure how should I leave this one as it also executes some code. Do we need it for correctness?

Reply:
Hey @adhorodyski 

Yes, this piece of code is a mandatory stuff in order to run test correctly. Basically what we are doing here is a subscription to an event, that report has been actually opened. If we receive this event, then we perform a navigation

Reply:
How do you see the exact steps that need to be taken here? https://github.com/Expensify/App/blob/7b9a0cd644ea36e1543ae63a23ea72f610029dc4/src/libs/E2E/tests/linkingTest.e2e.ts#L57

- On `sidebar_loaded` we trigger first navigation event, this stays as is: `Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(reportID));`
- **Question here**: where can I move the logic from `REPORT_INITIAL_RENDER`? Maybe I should trigger the navigation event to this linked message directly under the `sidebar_loaded` event?
- On `open_report` we submit the results, this stays as well

Reply:
> Maybe I should trigger the navigation event to this linked message directly under the sidebar_loaded event?

I think yes, but @perunt has implemented this test - maybe there was a specific reason why do we simulate two stage naviagation (maybe we wanted to precisely simulate user behavior?)

Reply:
@perunt could you please help me figure out this bit?

Reply:
As far as I remember, it was simulating navigation to a message within the same chat. That's why we first navigate to the chat (REPORT_INITIAL_RENDER) and then to the specific chat message (SWITCH_REPORT)

Reply:
1. This is already being reported inside of `chatOpeningTest.e2e.ts` (chat TTI)

Reply:
2. We don't measure time here, just assert it happened correctly. 

https://github.com/Expensify/App/blob/fd88517c7d50cf77cb400c007c1fc0bcf3be3d8b/src/pages/home/report/ReportActionsView.tsx#L422

If it doesn't remount, it won't call `open_report` for the 2nd time so we can't use it as a trigger to submit the result.

In this case I think we should navigate directly to the linked report on `sidebar_loaded`, right after the 1st navigation (to the main one).

Reply:
Pushed this now, @kirillzyusko can I ask you to test this on your end please? 🙏🏼

Reply:
Did we confirm this will work fine?

Reply:
Seems like @hannojg and @kirillzyusko confirmed this is working fine